lib/rest/Endpoints.js

```diff
@@ -3,6 +3,7 @@
 const {REST_VERSION} = require("../Constants");
 
 module.exports.BASE_URL = "/api/v" + REST_VERSION;
+module.exports.CLIENT_URL = "https://discordapp.com";
```

Comment:
```suggestion
module.exports.CLIENT_URL = "https://discord.com";
```

I would recommend just using the new format so this doesnt need to be changed later

Reply:
The domain is being changed in another PR. I like to stay consistent with the branch I'm currently in so I'd leave changing the domain up to someone else further down the line.

Reply:
The URL is going to change. Your PR is a new addition, your not modifying anything. It doesn't make sense to me to add something you know needs to be fixed in another PR.